src/Client.php

```diff
@@ -183,11 +183,12 @@ public function setHost($host): void
     /**
      * max_execution_time , in int value (seconds)
      *
+     * @param int|float $timeout
```

Comment:
**float** hint is kind of misleading here, because if caller file/class has `declare(strict_types=1)` then float would not be accepted as argument value

Reply:
Agreed, removed phpdocs since then they will just repeat function declaration.